tutorials/Native_distributions_and_local_execution/README.md

```diff
@@ -8,7 +8,10 @@ We will also demonstrate how to run an application locally with the same setting
 ## Gradle plugin
 
 `org.jetbrains.compose` Gradle plugin simplifies the packaging of applications into native distributions and running an application locally.
-Currently, the plugin uses [jpackage](https://openjdk.java.net/jeps/343) for packaging self-contained applications.
+Currently, the plugin uses [jpackage](https://openjdk.java.net/jeps/343) for packaging self-contained applications. Distributable 
+applications are self-contained, installable binaries which include all the Java runtime components they need, without requiring an installed 
```

Comment:
> packaging self-contained applications. Distributable applications are self-contained

Minor: I think mentioning, that applications "self-contained" twice is a bit excessive